examples/react-native-vanilla/package.json

```diff
@@ -24,6 +24,7 @@
     "@storybook/channels": "file:../../lib/channels",
     "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
     "@storybook/react-native": "file:../../app/react-native",
+    "@storybook/react": "file:../../app/react",
```

Comment:
Are you just adding this line to prove that the fix works, or does it make sense to have it for the RNV app?